app/assets/javascripts/num-players-slider.js

```diff
@@ -0,0 +1,73 @@
+let sliders = []
+
+document.addEventListener("turbolink:before-cache", function() {
+  if (sliders.length == 0) return;
+
+  sliders.forEach(slider => { slider.noUiSlider.destroy() });
+
+  sliders = []
+})
+
+document.addEventListener("turbolinks:load", function() {
+  const elements = document.querySelectorAll(".num-player-slider");
```

Comment:
For consistency please use a data attribute like `[data-role='num-player-slider']` (or `[data-num-player-slider]`). That way classes are strictly used for styling and data-attributes strictly used for javascript.